.circleci/config.yml

```diff
@@ -405,29 +473,121 @@ jobs:
                 docker tag "$IMAGE_NAME" "$IMAGE_NAME:$tag"
                 docker push "$IMAGE_NAME:$tag"
             done
+      - notify_slack_failure
       - notify_slack
 
+  deploy:
+    executor: deploy
+    steps:
+      - run:
+          name: Configure deploy secrets
+          command: |
+            if [ "$CIRCLE_PROJECT_USERNAME" != "omisego" ]; then
+              printf "\\033[0;33mSkipping deploy (running in fork)\\033[0;0m\\n"
+              exit 0
+            fi
+
+            GPGFILE=$(mktemp)
+            trap 'rm -f $GPGFILE' 0 1 2 3 6 14 15
+            echo "$CIRCLE_GPG_KEY" | base64 -d | gunzip > "$GPGFILE"
+            gpg --import "$GPGFILE"
+            printf "EB58C5A1AB6B04BEA2A6CFFFD93811C7530F9C72:6:\\n" | gpg --import-ownertrust
```

Comment:
Just a note to future self that this is the ID for the content in the `$GPGFILE` above. And it's the key that belongs to the CI so it can decrypt secrets for deployments.